google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/ConnectionWorker.java

```diff
@@ -623,6 +662,11 @@ private void requestCallback(AppendRowsResponse response) {
       }
       if (!this.inflightRequestQueue.isEmpty()) {
         requestWrapper = pollInflightRequestQueue();
+
+        Instant insertTime = requestWrapper.insertTime;
+        Instant currentTime = Instant.now();
+        long latencyToAdd = currentTime.toEpochMilli() - insertTime.toEpochMilli();
+        updateLatencyDistribution(latencyToAdd);
```

Comment:
Let's add the error message for now from client lib at L686. So in case there is still a blockage on customer's case, we will have more info to investigate.

Reply:
Do you mean the error message in response? I think that's already printed by Pablo's current code... I was planning to revert the change in this file once Pablo copy over this part

Reply:
Yes, we want it to be in the release we hand over to customers, so if anything is going on, we have some logs.

Reply:
let me clean this file up, Pablo has already copied the stats log, let's add error log directly

Reply:
But won't that kind of log overwhelmed the log console?